app/validators/preprint-form-validator.js

```diff
@@ -0,0 +1,38 @@
+import { validator, buildValidations } from 'ember-cp-validations';
+
+const Validations = buildValidations({
```

Comment:
That is true. For some reason when I was writing the validations I thought it would be useful to have it in a separate file in case I needed to import it in multiple places (which is what I had originally, but then realized it only needed to be used in the controller). I will delete the `preprint-form-validator.js` file and consolidate the validations into the controller.